src/commands/createEnvironment.ts

```diff
@@ -0,0 +1,40 @@
+import * as environment from '../utils/environment';
+import * as vscode from 'vscode';
+import * as shell from '../utils/shell';
+import { resolve } from 'path';
+
+export async  function buildLocalClusters() {	
+	await vscode.window.withProgress(
+		{
+		  location: vscode.ProgressLocation.Notification,
```

Comment:
This indentation on this line (and the following two lines too) doesn't seem right.